BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -362,7 +362,7 @@ extension AttributedLabel {
                     let style = textStorage.attribute(.paragraphStyle, at: 0, effectiveRange: &range)
 
                     guard let style = style as? NSParagraphStyle, range == textStorage.entireRange else {
-                        return .natural
+                        return textAlignment
```

Comment:
- [ ] Tests

Reply:
Does this actually return an non-default value? Since we are using the attributed string API for UILabel, we don't set this anywhere. I'm not sure it's a good fallback if we don't know where it's coming from.

Do we know why we're seeing this fail in the first place? If it's happening on a single centered line, like on your PR description, we should look into that closer.